Gemfile

```diff
@@ -2,7 +2,7 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gem "logstash-core", "2.0.0.dev", :path => "."
+gem "logstash-core", "2.0.0.snapshot1"
```

Comment:
@suyograo I would say "2.0.0.snapshot" since  "~> 2.0.0.beta" worked in my test to get the latest beta1...

I've looked through the rubygems doc this morning and I wasn't able to find any information about specifying version for prerelease gems, it might an unwanted effect? So after a bit of thinking maybe we should fix it with something we know ?